pipes/WDL/workflows/create_enterics_qc_viz.wdl

```diff
@@ -45,27 +49,35 @@ task create_viz {
         String          workspace_project
         String          input_table_name
 
-        String          grouping_column_name    = "gambit_predicted_taxon"
-        String          output_filename         = "QC_vizualizations.pdf"
+        String           grouping_column_name            =   "gambit_predicted_taxon"
+        String           output_filename                 =   "QC_visualizations.pdf"
+
+        String?         custom_est_coverage_thresholds
+        String?         custom_contig_thresholds
+        String?         custom_assembly_thresholds
+        String?         custom_mean_q_thresholds
 
-        String  docker                          =   "broadinstitute/horsefish:pgs_visualizations"        
+        String          docker                          =   "us-central1-docker.pkg.dev/pgs-automation/enterics-visualizations/create_visualization_html:v1"       
     }
 
     command {
-        python3 /scripts/create_visualizations.py -s ~{sep=' ' sample_ids} \
-                                                  -t ~{input_table_name} \
+        python3 /scripts/create_enterics_visualizations_html.py -s ~{sep=' ' sample_ids} \
+                                                  -dt ~{input_table_name} \
                                                   -w ~{workspace_name} \
-                                                  -p ~{workspace_project} \
+                                                  -bp ~{workspace_project} \
                                                   ~{"-g" + grouping_column_name} \
-                                                  ~{"-o" + output_filename}
+                                                  ~{"-o" + output_filename} \
+                                                  ~{"-ect" + custom_est_coverage_thresholds} \
+                                                  ~{"-cnt" + custom_contig_thresholds} \
+                                                  ~{"-at" + custom_assembly_thresholds} \
+                                                  ~{"-mqt" + custom_mean_q_thresholds} \
```

Comment:
can we just adjust the spacing on the parameters on lines 65-73 so that they line up with the `-s`